docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -1,90 +1,91 @@
+
+    !!! tip
+        If this is your first time using Codacy, see [how to add and analyze your first repository](./codacy-quickstart.md#adding-your-first-repository).
 
-    -   You have a [Codacy account](https://www.codacy.com/signup-codacy).
-    -   The repository you’re working on has been [added to Codacy Cloud](../organizations/managing-repositories.md#adding-a-repository) and analyzed.
-    -   You have at least [Repository Read permissions](../organizations/roles-and-permissions-for-organizations.md) for the repository you’re working on.
+1.  Install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=codacy-app.codacy) or through the Extensions view in VS Code.
 
-You can install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=codacy-app.codacy) or through the [Extensions view](https://code.visualstudio.com/docs/editor/extension-marketplace#_browse-for-extensions) in Visual Studio Code. Alternatively, you can [download the latest release as a VSIX package](https://github.com/codacy/codacy-vscode-extension/releases) and install it manually.
+    Alternatively, you can install it manually by [downloading the latest release as a VSIX package](https://github.com/codacy/codacy-vscode-extension/releases).
 
-## Fetching pull request quality and coverage data from Codacy
+## Getting pull request quality and coverage data
+
+To see Codacy quality and coverage data for the pull request you're working on, follow these steps:
 
 1.  Open the repository directory in VS Code.
 
     !!! note
-        Make sure that you’ve [added the repository to Codacy](../organizations/managing-repositories.md#adding-a-repository).
+        If the repository isn't on Codacy yet, [add it to Codacy](../organizations/managing-repositories.md#adding-a-repository) first.
 
 1.  Open the main view by clicking the **Codacy logo** in the status bar or the **Codacy tab** in the activity bar.
 
     ![Codacy main view](images/codacy-vscode-extension-sign-in.png)
 
 1.  If you’re not signed in, click the **Sign In** button to authorize VS Code on Codacy.
 
-1.  Check out the relevant branch.
+1.  Check out the branch you're working on.
 
     !!! note
-        The branch must match an open pull request analyzed by Codacy.
-
-After completing these steps, Codacy displays a summary of all found problems for the pull request in the main view and in VS Code's Problems tab.
-
-## Reviewing found problems
+        The branch must be associated with an open pull request analyzed by Codacy.
 
-To review a summary of all found problems for the pull request, open the main view by clicking the **Codacy logo** in the status bar or the **Codacy tab** in the activity bar.
+After completing these steps, Codacy displays a summary of all problems for the pull request in the main view and a list of found Quality issues in VS Code's Problems tab.
```

Comment:
Breaking the sentence for readability.
Also, maybe we can avoid "problems" by using "result of Codacy analysis"? (I heard this expression in the demo)

```suggestion
After completing these steps, the main view shows the result of the Codacy analysis for the pull request. The VS Code's Problems tab lists the Quality issues found.
```

Reply:
Looks good! Just a nitpick, removing the possessive `s`.

```suggestion
After completing these steps, the main view shows the result of the Codacy analysis for the pull request. The VS Code Problems tab lists the Quality issues found.
```